src/CommaSeparatedIntParser.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace MPScholten\RequestParser;
+
+class CommaSeparatedIntParser extends AbstractValueParser
+{
+    protected function parse($value)
+    {
+        if (empty($value)) {
+            return null;
+        }
+        $value = explode(',', $value);
+        return array_map('intval', $value);
```

Comment:
E.g. for input `1,2,somestring,3` this will return `[1, 2, 0, 3]`. IMO an exception should be thrown instead (this can be triggered by returning null instead of the array). The CommaSeparatedBooleanParser is already following this behavior and is returning null in case any of the values are invalid